### PR TITLE
Handle a workaround big.js ESM import

### PR DESCRIPTION
**Pull request by jstasiak**
Big.js can't be imported as-is in ESM mode[1] but there's a workaround:
it can be imported as 'big.js/big.mjs'. The unfortunate downside of this
is the lack of type coverage, which this patch aims to rectify.

[1] https://github.com/MikeMcl/big.js/issues/170

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/MikeMcl/big.js/issues/170

### CODE REVIEW

**Comment by jstasiak** [comment]
CI failed for unrelated reason, errors present in the `master` branch: https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=131089&view=logs&j=8635df7d-ecde-52a7-c7a8-84b998e35690&t=c264d665-0b79-59b7-7e04-9965cbe74860

File: types/big.js/big.mjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export * from './';
+import Big from '/';
```
Comment (amcasey) [inline types/big.js/big.mjs.d.ts:2]: Typo?  It's unusual to import from an absolute path.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/big.js/big.mjs.d.ts (hunk 1 above)
Comment (jstasiak) [reply types/big.js/big.mjs.d.ts]: You're absolutely right, fixed

**Comment by amcasey** [comment]
To be honest, this seems like a bit of a hack.  Having said that, it's modelling a hack in the underlying library, so it's probably okay?   Thoughts, @nycdotnet @Ky6uk?

**Comment by jstasiak** [comment]
For what it's worth big.js 6.2.0 has been released and it can be imported in ESM code just fine now so I'll close this PR.

**Pull request closed without merging**